homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -293,6 +296,11 @@ def fan_speed_list(self):
         """Get the list of available fan speed steps of the vacuum cleaner."""
         return list(self._fan_speeds)
 
+    @property
+    def timers(self):
+        """Get the list of added timers of the vacuum cleaner."""
+        return [{"enabled": timer.enabled, "time": timer.ts} for timer in self._timers]
```

Comment:
What kind of time is `timer.ts`? Is it relative or absolute time? Local or utc?

We only allow absolute utc time in state attributes.

Reply:
That is actually just the internal id (creation time) of the given timer, in vacuum's local time I suppose. I'm not sure if that's actually so useful for users? @MarBra could you describe your use-case for this a bit?

Reply:
Oh yes that is of course not useful for users :( Sorry for that.
I will push a new commit with the cron formatted string and will add a key for the next schedule in utc time.

Reply:
The use-case is to display the scheduled runs in the UI.